load_balancer.tf

```diff
@@ -17,13 +20,8 @@ resource "aws_elb" "reutlb" {
     target              = "HTTP:80/"
     interval            = 30
   }
-  instances                   = aws_instance.reut_db.*.id
-  cross_zone_load_balancing   = true
-  idle_timeout                = 400
-  connection_draining         = true
-  connection_draining_timeout = 400
 
```

Comment:
I deleted all of this because it was not part of the homework and most of the entries are irrelevant here.
For `instances` field, I changed to the correct syntax, as I showed above.

Reply:
but what about the db instances? how do i heakth check them?

Reply:
helah check for what?
These are just instances without any lb.